Sources/NautilusTelemetry/Tracing/Baggage.swift

```diff
@@ -8,6 +8,17 @@
 import Foundation
 import os
 
+// MARK: - SubtraceLinking
+
+public enum SubtraceLinking {
+	case none // don't link subtraces
+	case up // link from child to parent
+	case down // link from parent to child
+	case bidirectional // link both directions
+}
```

Comment:
Another way to model this would be with an `OptionSet` (e.g. `[.up]` or `[.up, .down])`).